src/EditorFeatures/CSharpTest/Diagnostics/GenerateEnumMember/GenerateEnumMemberTests.cs

```diff
@@ -130,6 +130,14 @@ public class GenerateEnumMemberTests : AbstractCSharpDiagnosticProviderBasedUser
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEnumMember)]
+        public async Task TestBinary()
+        {
+            await TestAsync(
+@"class Program { void Main ( ) { Color . [|Blue|] ; } } enum Color { Red = 0b01 } ",
+@"class Program { void Main ( ) { Color . Blue ; } } enum Color { Red = 0b01 , Blue = 0b10 } ");
```

Comment:
What happens if `Red = 0b11`?
The code below seems to be hardcoded to substring of 2 characters, but I think binary literals allow more than 2 bits, right?